### PR TITLE
Ep/sidebargen

### PR DESCRIPTION
**Pull request by ErikApption**
Added new tool to generate sidebars

### CODE REVIEW

File: Shared/src/Datahub.Shared/Datahub.Shared.projitems [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Entities\TerraformWorkspace.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Entities\TerraformTemplate.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Enums\TerraformRole.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Exceptions\ProjectNoFoundException.cs" />
+    <Compile Include="..\..\..\Shared\src\Datahub.Shared\Exceptions\ProjectNotFoundException.cs" />
```
Comment (yjmrobert) [inline Shared/src/Datahub.Shared/Datahub.Shared.projitems]: This doesn't feel right @ErikApption, was that added by VS?

Comment (yjmrobert) [reply Shared/src/Datahub.Shared/Datahub.Shared.projitems]: `<Compile Include="$(MSBuildThisFileDirectory)Exceptions\ProjectNoFoundException.cs" />`

**Review by yjmrobert — CHANGES_REQUESTED** [review]

File: Shared/src/Datahub.Shared/Exceptions/ProjectNotFoundException.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Datahub.Shared.Exceptions;
+
+public class ProjectNotFoundException : Exception
+{
+    public ProjectNotFoundException(string message) : base(message)
+    {
+    }
+}
```
Comment (yjmrobert) [inline Shared/src/Datahub.Shared/Exceptions/ProjectNotFoundException.cs:10]: image
Hahaha, can we keep just the one?

File: Portal/test/Datahub.ProjectTools.Tests/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (iroelperez) [inline Portal/test/Datahub.ProjectTools.Tests/Properties/launchSettings.json:1]: Why there is new launchSettings in many projects we never launch? :)
No big deal tho, just curious.

**Review by iroelperez — APPROVED** [review]

File: Shared/src/Datahub.Shared/Datahub.Shared.projitems (hunk 1 above)
Comment (ErikApption) [reply Shared/src/Datahub.Shared/Datahub.Shared.projitems]: fixed

**Pull request merged**